### PR TITLE
Adds a no-welding version of the four-winds shotgun.

### PR DESCRIPTION
**Pull request by ghost**
#### Summary

SUMMARY: Content "Adds a no-welder version of the four winds shotgun"

#### Purpose of change

I promised myself I would add this. So I did. Enjoy your new early, early-game weapon. You need a pair of pliers, a makeshift hammer, a hand drill and a filing tool.

#### Describe the solution

This is an easier to make version of the slam-fire shotgun, also known as the four winds shotgun. I wanted players to experience pipe weapons again.

#### Describe alternatives you've considered

None. If I hadn;t wanted this in this video game, I would not have done it.

#### Additional context

I plan to change the shotgun traps to use these four winds shotguns. Currently, they're not used for anything.

A double barrel version might come soon. It's just two of these things taped together, easy as. Mechanics 0.

Also, y'all might want to keep an eye out or a new underwater-usable weapon incorporating this... coming Soon(TM)

### CODE REVIEW

File: data/json/recipes/weapon/ranged.json [hunk 1]
```diff
@@ -301,5 +301,49 @@
...
+  },
+  {
+    "type": "recipe",
+    "id_suffix": "without_welding",
+    "result": "four_winds_shotgun",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_RANGED",
+    "skill_used": "mechanics",
+    "skills_required": [ [ "gun", 1 ] ],
+    "difficulty": 0,
```
Comment (LaVeyanFiend) [inline data/json/recipes/weapon/ranged.json:313]:
```suggestion
    "skill_used": "mechanics",
    "skills_required": [ [ "gun", 1 ] ],
```

File: data/json/items/gun/shot.json [hunk 2]
```diff
@@ -827,10 +827,10 @@
     "pocket_data": [ { "pocket_type": "MAGAZINE", "ammo_restriction": { "shot": 6 } } ]
   },
   {
-    "id": "slam_shotgun",
+    "id": "four_winds_shotgun",
```
Comment (LaVeyanFiend) [inline data/json/items/gun/shot.json:830]: You'd need a migration entry to json/items/migration.json if you're going to change the ID.
```
{
    "id": "slam_shotgun",
    "type": "MIGRATION",
    "replace": "four_winds_shotgun"
  },
```

File: data/json/recipes/weapon/ranged.json [hunk 3]
```diff
@@ -260,13 +260,13 @@
   },
   {
     "type": "recipe",
-    "result": "slam_shotgun",
+    "result": "four_winds_shotgun",
```
Comment (LaVeyanFiend) [inline data/json/recipes/weapon/ranged.json:263]: You'd need to add a recipe obsoletion to json/recipes/recipe_obsolete.json to change the ID:
```
{
    "type": "recipe",
    "result": "slam_shotgun",
    "obsolete": true
  },
```

File: data/json/items/gun/shot.json (hunk 2 above)
Comment (Xenomorph-III) [reply data/json/items/gun/shot.json:830]: Er, WHAT?
Wow, K.

File: data/json/recipes/weapon/ranged.json (hunk 3 above)
Comment (Xenomorph-III) [reply data/json/recipes/weapon/ranged.json:263]: Where do I put these things?

File: data/json/items/migration.json [hunk 4]
```diff
@@ -1233,5 +1233,10 @@
     "id": "carton_unsealed",
     "type": "MIGRATION",
     "replace": "carton_sealed"
-  }
+  },
+  {
+    "id": "slam_shotgun",
+    "type": "MIGRATION",
+    "replace": "four_winds_shotgun"
+  },
```
Comment (LaVeyanFiend) [inline data/json/items/migration.json:1241]: No comma

File: data/json/recipes/recipe_obsolete.json [hunk 5]
```diff
@@ -2499,5 +2499,10 @@
     "type": "recipe",
     "result": "bone_plate",
     "obsolete": true
-  }
+  },
+  {
+    "type": "recipe",
+    "result": "slam_shotgun",
+    "obsolete": true
+  },
```
Comment (LaVeyanFiend) [inline data/json/recipes/recipe_obsolete.json:2507]: Comma at end bad

**Pull request merged**